src/WebJobs.Script/Workers/Http/DefaultHttpWorkerService.cs

```diff
@@ -214,5 +214,18 @@ private async Task<bool> IsWorkerReadyForRequest()
                 return false;
             }
         }
+
+        public async Task PingAsync()
+        {
+            string requestUri = new UriBuilder(WorkerConstants.HttpScheme, WorkerConstants.HostName, _httpWorkerOptions.Port).ToString();
+            try
+            {
+                await _httpClient.GetAsync(requestUri);
```

Comment:
Code here: https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/Workers/Http/DefaultHttpWorkerService.cs#L201 is used for initial ping. Can we use the same? Refactor ?

Reply:
Thanks. refactored a bit around that.